Form-Controls/styles.css

```diff
@@ -0,0 +1,78 @@
+/* * {
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+} */
+html {
+  background-color: #eee;
+  display: flex;
+  justify-content: center;
+}
+
+header {
+  text-align: center;
+}
+
+main {
+  display: flex;
+  justify-content: center;
+}
+
+body {
+  background-color: white;
+  width: 500px;
+  margin-top: 4em;
+}
+
+form {
+  display: grid;
+}
+
+fieldset {
```

Comment:
Try to target classes rather than elements, then if one of them needs to look different, it's easy to do and you don't have to 'undo' the styling that was applied to all elements.